### DISCUSSION TITLE
Explanation of solve in API

### DISCUSSION THREAD

**Discussion by Peter230655** [csu-hmc/opty#222] [General]
In the latest API it states:
![image](https://github.com/user-attachments/assets/0fb72eb9-d772-490a-9e17-47064b411acf)
An explanation for **x** is given, but not for Lagrange = [], zu = [], zl = [].
I could not find zu or zl anywhere else in the API.
Since I have used solve(...) I know they are not needed, but maybe some explanation would be helpful?

**Comment by moorepants** [comment]
This should be opened on the cyipopt repository.

**Reply by Peter230655** [reply]
> This should be opened on the cyipopt repository.

I just did so, so I guess I can close this?
